book/website/community-handbook/community-calls.md

```diff
@@ -0,0 +1,63 @@
+(ch-community-calls)=
+# _The Turing Way_ Community Calls
```

Comment:
@richarddushime - Maybe we can add information here about joining the community calls in low-bandwidth environments?

Reply:
Great , I will push in  a commit